<launch>
  <!-- (start your robot's MoveIt! stack, e.g. include its moveit_planning_execution.launch) -->
  <!-- (start your tracking system's ROS driver) -->


  <!-- start ArUco -->
    <node name="aruco_tracker" pkg="aruco_ros" type="single">
        <remap from="/camera_info" to="/camera/rgb/camera_info" />
        <remap from="/image" to="/camera/rgb/image_raw" />
        <param name="image_is_rectified" value="true"/>
        <param name="marker_size"        value="0.088"/>
        <param name="marker_id"          value="300"/>
        <!-- <param name="reference_frame"    value="camera_link"/> -->
        <param name="camera_frame"       value="camera_rgb_optical_frame"/>
        <param name="marker_frame"       value="camera_marker" />
    </node>


  <include file="$(find easy_handeye)/launch/calibrate.launch">
    <arg name="eye_on_hand" value="false"/>
    <!-- <arg name="namespace" value="my_eob_calib"/> -->

    <!-- fill in the following parameters according to your robot's published tf frames -->
    <arg name="robot_base_frame" value="base_link"/>
    <arg name="robot_effector_frame" value="ee_link"/>

    <!-- fill in the following parameters according to your tracking system's published tf frames -->
    <!-- kinect fames -->
    <arg name="tracking_base_frame" value="camera_link"/>
    <arg name="tracking_marker_frame" value="camera_marker"/>
    
    <arg name="freehand_robot_movement" value="true"/>
    <arg name="start_rviz" value="false"/>
  </include>
</launch>